Configurations/10-main.conf

```diff
@@ -1557,6 +1557,14 @@ my %targets = (
         bn_ops           => "SIXTY_FOUR_BIT_LONG",
         perlasm_scheme   => "macosx",
     },
+    "darwin64-arm64-cc" => {
+        inherit_from     => [ "darwin-common", asm("aarch64_asm") ],
+        CFLAGS           => add("-Wall"),
+        cflags           => add("-arch arm64"),
+        lib_cppflags     => add("-DL_ENDIAN"),
+        bn_ops           => "SIXTY_FOUR_BIT_LONG",
+        perlasm_scheme   => "ios64",
```

Comment:
Curiosity question for my own self interest, why is the perlasm_scheme listed as ios64?

Reply:
Because it is the same assembly used for iOS 64-Bit devices. The M1 is basically an evolution of the A-Series chips used in iPhones and iPads.